### PR TITLE
Switch to conda-forge managing Python and R environments

### PR DESCRIPTION
**Pull request by davidhemp**
Due to the changes in the Anaconda ToS we would like to direct users to conda-forge. Anaconda did back track a little from the original changes but it is still possible to see a researcher that has an corporate sponsor as needing a license to use Anaconda. From https://www.anaconda.com/pricing/terms-of-service-faqs

_Anaconda is free for all educational entities—even those with 200 or more employees or contractors—when used in a course curriculum, including teaching, learning, and researching, at accredited universities worldwide. 

A paid license is always required for any individual or entity—even an educational entity—that is embedding, mirroring, or providing third parties access to our products. If you’re interested in pursuing an embedded use case, please contact our partner team at [partnerships@anaconda.com](mailto:partnerships@anaconda.com). See the [Anaconda Terms of Service](https://legal.anaconda.com/policies/en/) for details_

### CODE REVIEW

File: docs/hpc/applications/guides/openondemand.md
Comment (dithwick) [inline docs/hpc/applications/guides/openondemand.md:1]: Using miniconda won't work with this until the OOD changes have been merged into production.

File: docs/hpc/applications/guides/tensorflow.md
Comment (dithwick) [inline docs/hpc/applications/guides/tensorflow.md:1]: There's a reference to Anaconda on line 5.

**Review by dithwick — CHANGES_REQUESTED** [review]
Mostly good. Number of small changes. Also the OOD changes won't work until the change has been pushed to production but that will be soon so doesn't necessarily have to hold this up.

File: docs/hpc/applications/guides/tensorflow.md
Comment (davidhemp) [reply docs/hpc/applications/guides/tensorflow.md:1]: Switched to "conda"

File: docs/hpc/applications/guides/openondemand.md
Comment (davidhemp) [reply docs/hpc/applications/guides/openondemand.md:1]: Did you want to delay until then for merge and let OOD catch up?

Comment (dithwick) [reply docs/hpc/applications/guides/openondemand.md:1]: What about pulling the openondemand.md changes out of this PR and into another one? That way this can be merged ASAP.

**Review by dithwick — APPROVED** [review]
Looks good.

File: docs/hpc/applications/guides/openondemand.md
Comment (davidhemp) [reply docs/hpc/applications/guides/openondemand.md:1]: Changed it back to 7f82cff3ac3395ec429a61fb4ddf5bb4dba76501

**Pull request merged**
